search/searcher/search_fuzzy.go

```diff
@@ -71,6 +84,21 @@ func NewFuzzySearcher(ctx context.Context, indexReader index.IndexReader, term s
 		boost, options, true)
 }
 
+func getAutoFuzziness(term string) int {
+	termLength := len(term)
+	if termLength > 5 {
+		return 2
+	} else if termLength > 2 {
+		return 1
+	}
```

Comment:
Can you add some commentary around here on why we're fixing these at these numbers?
I think it'd be better to add global exported vars for these constants allowing library users to tweak them if necessary.

Reply:
done